spec/shared_contexts/l10n.rb

```diff
@@ -5,6 +5,7 @@
     @old_locale = Locale.current
     Locale.current = locale
     Puppet::GettextConfig.setup_locale
+    Puppet::GettextConfig.create_default_text_domain
```

Comment:
`Puppet::GettextConfig.instance_variable_get(:@gettext_disabled)` is now false when the `before(:all)` block is called. Likely due to the call to initialize_settings now explicitly calling `Puppet::GettextConfig.disable_gettext` instead of when the hook was called?